<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QMainWindow" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>91</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="board_3" rowstretch="0,0,0" columnstretch="0">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="2" column="0">
      <widget class="QPushButton" name="button_8">
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="button_5">
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="button_2">
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="board_2" rowstretch="0,0,0" columnstretch="0">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="2" column="0">
      <widget class="QPushButton" name="button_7">
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="button_4">
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="button_1">
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>91</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="board_1" rowstretch="0,0,0" columnstretch="0">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="2" column="0">
      <widget class="QPushButton" name="button_9">
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="button_6">
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="button_3">
       <property name="font">
        <font>
         <pointsize>40</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Who's turn:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="whos_turn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Tlwg Typo</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Winner:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="winner">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
